garrysmod/gamemodes/base/entities/entities/npc_tf2_ghost.lua

```diff
@@ -66,7 +66,7 @@ end
 -- List the NPC as spawnable
 --
 list.Set( "NPC", "npc_tf2_ghost", {
-	Name = "Example NPC",
+	Name = "#npc_example",

```

Comment:
This will not work. There is a reason why all localization strings for entities' names are using their ID names.

For example, the kill feed will try to find `#npc_tf2_ghost` and will ignore `#npc_example`, since the ID for that Example NPC is `npc_tf2_ghost`.

![image](https://github.com/user-attachments/assets/9b607880-2b83-4f0e-a2fd-a502efca9008)